src/circuits/etable_compact/op_configure/op_load.rs

```diff
@@ -44,6 +45,14 @@ pub struct LoadConfig {
     is_eight_bytes: BitCell,
     is_sign: BitCell,
 
+    highest_bit: BitCell,
+    is_zero_byte_padding: BitCell,
```

Comment:
The loading size is `1, 2, 4, 8`, the stack value size is `4, 8`, so the options are:
```
(1  |  2  |  4), 8  -> bytes(4..7) is extended
(2), _ -> bytes(2..3) is also extended
(1), _ -> bytes(1) is also extended
```

Reply:
nice! Removed those bits.